isamples_metadata/GEOMETransformer.py

```diff
@@ -524,6 +524,10 @@ def parse_permit_freetext(text: str) -> dict[str, list[str]]:
 
         return GEOMETransformer._format_result_object([original_string])
 
```

Comment:
This was a weirdness I found in the importer when turning on code coverage and running `pytest --cov` locally.  It was upset about `geo_to_h3` being in the module and was failing on the import.  The workaround was to move it inside the class called `GEOMETransformer` in the `GEOMETransformer` module.  This was some of the first python I wrote and didn't understand the distinction between classes and modules.  A long term fix would be to rename all the `XXXTransformer.py` to something like `xxx_transformer.py` in line with python recommended naming.